documentation/index.html.md

```diff
@@ -31,6 +31,7 @@ The epicenter.js library is a set of services (adapters) and managers to help st
 	* [World Adapter](./generated/world-api-adapter/)
 
 * The following managers add functionality on top of their respective services:
+	* [Authorization Manager](./generated/auth-manager/): The Authorization Manager provides an easy way to track and access user authentication (logging in and out) and authorization (keeping track of tokens, sessions, and groups) for projects.
```

Comment:
It's not _just_ tracking though, it's actually doing those things. "..easy way to *manage* user authentication (logging in and out) and authorization (keeping track of tokens, sessions, and groups) across projects."?